apps/content-gateway-api/src/app/service/v1/rest/ContentGatewayAPI.ts

```diff
@@ -39,25 +60,210 @@ export const generateContentGatewayAPIV1 = async ({
 
     const router = express.Router();
 
-    router.get("/schema/stats", async (_, res) => {
+    const extractUser = (
```

Comment:
This is how we figure out the user from the supplied API key. If there is no API key we assume an anon user. Anons have no permissions yet, but later we can improve this (for example the GraphQL API is not authorized at all right now).